Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,232 @@
+***
+
+## Running the server
+
+In order to run a local server one must first locally clone the [Read-write-web play repository](https://github.com/read-write-web/rww-play). After that the user must navigate to their local Rww-play directory and run git there. For more information on how to run the server and the exact command used refer to the [Rww-play README page](https://github.com/read-write-web/rww-play).
+
+Users can also publish on their local Read-write-web play. In order to do this, a user must run the following commands inside the local rww-play directory.
+
+```bash
+$ git 
+> publishLocal
+```
+
+One can then import it into their scripts from inside the ammonite shell. The user should then be able to make a get on their local rww-play and check whether they can access a resource that has access control. This means that the user will need to make themselves a key.
+
+## The Key Pair
+
+A key pair is a pair formed from a Public and a Private key ----------- (more info on public and private keys ?)
+
+### Resolving dependencies
+
+In order to create this pair, the user must first import several dependencies from inside their ammonite shell.
+
+```scala
+@ import coursier.core.Authentication, coursier.MavenRepository
+
+@ interp.repositories() ++= Seq(MavenRepository(
+     "http://bblfish.net/work/repo/snapshots/"
+  ))
+
+@ import $ivy.`org.w3::banana-jena:0.8.5-SNAPSHOT`
+@ import org.w3.banana._
+@ import org.w3.banana.syntax._
+@ import org.w3.banana.jena.Jena
+@ import Jena._
+@ import Jena.ops._
+```
+
```

Comment:
In the original introduction to scripting with ammonite I showed the `@` sign in order to help the new user see the difference betwen a command in ammonite and a response from the shell. But it is not very helpful to show those when the response is not important, as it makes copying and pasting the code into a shell difficult.